cmd/plan_test.go

```diff
@@ -88,3 +89,45 @@ func (suite *PlanTestSuite) TestGetPriorInstalledPackages_NoPreinstalledPackages
 func installedPackagesAreEqual(expected, actual desc.Desc) bool {
 	return expected.Package == actual.Package && expected.Version == actual.Version && expected.Repository == actual.Repository
 }
+
+func CopyDir(fs afero.Fs, src string, dst string) error {
+
+	err := fs.MkdirAll(dst, 0755)
+	if err != nil {
+		return err
+	}
+
+	openedDir, err := fs.Open(src)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		openedDir.Close() //If we
```

Comment:
rather than deferring, especially since this is a recursively called function, this is an example where we should likely explicitly close as soon as we're done, which looks like after readdir in like 108